### PR TITLE
Opponent Play

### PR DESCRIPTION
**Pull request by rodney528**
I noticed opponent play was commented option in ClientPrefs so I'm doing this now ig.

### CODE REVIEW

**Comment by antoSKRLOL** [comment]
finally someone fixed it (I hope so)

File: source/PlayState.hx [hunk 1]
```diff
@@ -1313,7 +1315,9 @@ class PlayState extends MusicBeatState
 	
 		#if desktop
 		// Updating Discord Rich Presence.
-		DiscordClient.changePresence(detailsText, SONG.song + " (" + storyDifficultyText + ")", iconP2.getCharacter());
+		var setIcon:HealthIcon = iconP2;
+		if (opponentPlay) {setIcon = iconP1;} else {setIcon = iconP2;}
```
Comment (UmbratheUmbreon) [inline source/PlayState.hx:1319]: You could just do 
`var setIcon:HealthIcon = (opponentPlay ? iconP1 : iconP2);`

**Comment by DillyzThe1** [comment]
hey i've actually done this in both kade and psych engine before (both times were a result of leftover code i had from making online multiplayer for my friends) so if you have any problems getting anything to work let me know

**Comment by ImaginationSuperHero528** [comment]
> hey i've actually done this in both kade and psych engine before (both times were a result of leftover code i had from making online multiplayer for my friends) so if you have any problems getting anything to work let me know

Alr thx

File: source/PlayState.hx (hunk 1 above)
Comment (ImaginationSuperHero528) [reply source/PlayState.hx:1319]: Oh... I didn't know that so thx!

**Review by Luisinhi010 — APPROVED** [review]

**Comment by RodneyAnImaginativePerson** [comment]
> finally someone fixed it (I hope so)

Wdym fixed?

**Comment by RodneyAnImaginativePerson** [comment]
JESUS

**Comment by RodneyAnImaginativePerson** [comment]
I didn't think everytime I saved pushed my damn work!

**Comment by RodneyAnImaginativePerson** [comment]
For y'all wondering I was coding on my phone lol

**Review by charlesisfeline — APPROVED** [review]

**Comment by RodneyAnImaginativePerson** [comment]
[VIDEO](https://www.youtube.com/watch?v=lG9UakC4c1I) :D

**Comment by RodneyAnImaginativePerson** [comment]
Might move to the experimental branch!

**Review by charlesisfeline — CHANGES_REQUESTED** [review]
you forgot something

```source/PlayState.hx:4757: characters 2-8 : Expected }```

**Review by charlesisfeline — CHANGES_REQUESTED** [review]
you forgot something

```source/PlayState.hx:4757: characters 2-8 : Expected }```

the compiler is showing a compiler error from a completely unrelated line

**Comment by RodneyAnImaginativePerson** [comment]
> you forgot something
> 
> `source/PlayState.hx:4757: characters 2-8 : Expected }`
> 
> the compiler is showing a compiler error from a completely unrelated line

idk whats wrong give me 20 years

**Comment by doom699** [comment]
time to play opposition and break the fingers

**Comment by RodneyAnImaginativePerson** [comment]
HEY I NEED TO PORT TO 0.7 STILL >:(

**Comment by RodneyAnImaginativePerson** [comment]
> > HEY I NEED TO PORT TO 0.7 STILL >:(
> 
> he closed 99% of pre-0.7 PRs

Ah

**Pull request closed without merging**